changelogs/fragments/add_argument_check_for_rundeck.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- rundeck_acl_policy - Add check for rundeck_acl_policy name parameter
```

```suggestion
- rundeck_acl_policy - add check for rundeck_acl_policy name parameter (https://github.com/ansible-collections/community.general/pull/612).
```
according to the doc